README.md

```diff
@@ -72,12 +72,17 @@ The server is configured with command-line options. See
 
 The `--listen` option specifies the interface and port the server listens on.
 It must contain an IP-Address or a DNS name and a port number. This option is
-mandatory, but can be repeated to specify multiple interfaces or ports.
+mandatory, but can be repeated to specify multiple interfaces or ports. This
+value can be specified in environment variable `LISTEN`, as a comma-separated
+list of values.
 
-The `--data-dir` option specifies where the server should store its data.
+The `--data-dir` option specifies where the server should store its data. This
+value can be specified in the environment variable `DATA_DIR`.
 
 By default, the server allows all client IDs. To limit the accepted client IDs,
 such as when running a personal server, use `--allow-client-id <client-id>`.
+This value can be specified in the environment variable `CLIENT_ID`, as a
+comma-separated list of client IDs.
```

Comment:
Might be nice to mention that the client id should be treated as a credential since it may not be obvious that the client id is the only authentication.